pkg/api/registered/registered.go

```diff
@@ -25,71 +26,97 @@ import (
 	"k8s.io/kubernetes/pkg/api/unversioned"
 )
 
-// List of registered API versions.
-// The list is in the order of most preferred to the least.
-var RegisteredGroupVersions []unversioned.GroupVersion
+var (
+	// registeredVersions represents all enabled API versions. Please call
+	// RegisterVersions() to register the versions.
+	registeredVersions = map[unversioned.GroupVersion]struct{}{}
```

Comment:
Note in the last commit, I ran a grep/sed to `s/enabled/registered`, so logically all `registeredXXX` variables in this PR represent `enabledXXX`. And there is no variable represent the concept of registered versions in this PR.

Sorry for the confusion.